BUILDING.md

```diff
@@ -15,6 +15,15 @@ or MonoDevelop.
 
 The solutions all place their output in a common bin directory under the solution root.
 
+## Running Tests
+
+The tests that should be run in the solution are grouped by project name:
+ * `nunit.framework.tests-*`
+ * `nunitlite.tests-*`
+ * `slow-nunit-tests-*`
```

Comment:
Sorry to have missed this PR - I was still back on the issue, asking questions, because I didn't understanding what problem we were trying to solve

Anyway, this is wrong. The `slow-nunit-tests` assemblies are not intended to be run. They are merely data for our actual tests to use.